frontend/src/store/reducers/userReducers.js

```diff
@@ -1,70 +1,72 @@
-import {
-	REGISTER_USER,
-	LOGIN_USER,
-	LOGOUT_USER,
-	FETCH_USER_DATA,
-	USER_DATA_FETCHED,
-	USER_ERROR
-} from '../actions';
-
```

Comment:
Why remove the importing of variables from actions? Defining the variable as a string and using the variable between actions and reducers cuts down on typos